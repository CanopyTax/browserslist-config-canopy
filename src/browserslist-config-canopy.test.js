const path = require('path')
const browserslist = require('browserslist')
const queries = require('./browserslist-config-canopy.js')

describe('browserslist-config-canopy', () => {
  it(`exports a valid browserslist config that is what we want for Canopy`, () => {
    const browsers = browserslist(queries)
    expect(browsers).toMatchSnapshot()
  })
})
