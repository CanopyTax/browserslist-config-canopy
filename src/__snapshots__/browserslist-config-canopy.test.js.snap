// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`browserslist-config-canopy exports a valid browserslist config that is what we want for Canopy 1`] = `
Array [
  "chrome 121",
  "chrome 120",
  "chrome 119",
  "chrome 118",
  "chrome 117",
  "chrome 116",
  "chrome 115",
  "edge 121",
  "edge 120",
  "edge 119",
  "firefox 122",
  "firefox 121",
  "firefox 120",
  "firefox 119",
  "firefox 118",
  "ios_saf 17.3",
  "ios_saf 17.2",
  "ios_saf 17.1",
  "ios_saf 17.0",
  "ios_saf 16.6-16.7",
  "ios_saf 16.5",
  "ios_saf 16.4",
  "ios_saf 16.3",
  "ios_saf 16.2",
  "ios_saf 16.1",
  "ios_saf 16.0",
  "ios_saf 15.6-15.8",
  "ios_saf 15.5",
  "ios_saf 15.4",
  "ios_saf 15.2-15.3",
  "ios_saf 15.0-15.1",
  "opera 106",
  "opera 105",
  "opera 104",
  "safari 17.3",
  "safari 17.2",
  "safari 17.1",
  "safari 17.0",
  "safari 16.6",
  "safari 16.5",
  "safari 16.4",
  "safari 16.3",
  "safari 16.2",
  "safari 16.1",
  "safari 16.0",
  "safari 15.6",
  "safari 15.5",
  "safari 15.4",
  "safari 15.2-15.3",
  "safari 15.1",
  "safari 15",
]
`;
